src/services/note/delete.ts

```diff
@@ -64,4 +69,30 @@ export default async function(user: User, note: Note, quiet = false) {
 			});
 		}
 	}
+
+	await Notes.delete({
+		id: note.id,
+		userId: user.id
+	});
+}
+
+async function findCascadingNotes(note: Note) {
+	const cascadingNotes: Note[] = [];
+
+	const recursive = async (noteId: string) => {
+		const replies = await Notes.find({ replyId: noteId });
```

Comment:
It would be better to get `user` with JOIN.
```suggestion
		const query = Notes.createQueryBuilder('note')
			.where('note.replyId = :noteId', { noteId })
			.leftJoinAndSelect('note.user', 'user');

		const replies = await query.getMany();
```

Reply:
Fixed :>